deep-learning/_posts/2017-08-31-egomotion.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Unsupervised Learning of Depth and Ego-Motion from Video"
+tags: deep-learning CNN RNN
```

Comment:
Not sure there are RNN in the paper.  Also, you should put "video" and "Depth" as tags (and add those tags in the howto file)